### DISCUSSION TITLE
Firebase App Check token is invalid.

### DISCUSSION THREAD

**Discussion by Lumca** [vuejs/vuefire#1475] [Questions and Help]
Did anyone encounter:
```
{
  "error": {
    "code": 401,
    "message": "Firebase App Check token is invalid."
  }
}
```

I am trying to get data from FB storage.

I tried app check with reCAPTCHA v3 and enterprise. Then enforce lock on storage.

I can login to the app no issue, but when retriving data. it fails on this.
I have setup secret key in firebase and the client side key in nuxt.

otherwise, all seems to work great so far. 👍🏻

**Comment by posva** [comment]
Did you check https://vuefire.vuejs.org/nuxt/deployment.html examples? You can probably compare your code to that
